config/jobs/etcd/etcd-periodics.yaml

```diff
@@ -598,3 +598,331 @@ periodics:
             memory: "3Gi"
     nodeSelector:
       kubernetes.io/arch: arm64
+
+- name: ci-etcd-e2e-release36-arm64
+  interval: 4h
+  cluster: k8s-infra-prow-build
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+    - org: etcd-io
+      repo: etcd
+      base_ref: release-3.6
+  annotations:
+    testgrid-dashboards: sig-etcd-periodics, sig-etcd-arm64
+    testgrid-tab-name: ci-etcd-e2e-release36-arm64
+  spec:
+    containers:
+    - image: us-central1-docker.pkg.dev/k8s-staging-test-infra/images/kubekins-e2e:v20250205-f1f3519e6b-master
+      command:
+      - runner.sh
+      args:
+      - bash
+      - -c
+      - |
+        set -euo pipefail
+        VERBOSE=1 GOOS=linux GOARCH=arm64 CPU=4 EXPECT_DEBUG=true make test-e2e-release
+      resources:
+        requests:
+          cpu: "4"
+          memory: "8Gi"
+        limits:
+          cpu: "4"
+          memory: "8Gi"
+    nodeSelector:
+      kubernetes.io/arch: arm64
+
+- name: ci-etcd-e2e-release36-amd64
+  interval: 4h
+  cluster: eks-prow-build-cluster
+  decorate: true
+  decoration_config:
+    timeout: 60m
+  extra_refs:
+    - org: etcd-io
+      repo: etcd
+      base_ref: release-3.6
+  annotations:
+    testgrid-dashboards: sig-etcd-periodics, sig-etcd-amd64
+    testgrid-tab-name: ci-etcd-e2e-release36-amd64
+  spec:
+    containers:
+    - image: us-central1-docker.pkg.dev/k8s-staging-test-infra/images/kubekins-e2e:v20250205-f1f3519e6b-master
+      command:
+      - runner.sh
+      args:
+      - bash
+      - -c
+      - |
+        set -euo pipefail
+        VERBOSE=1 GOOS=linux GOARCH=amd64 CPU=4 EXPECT_DEBUG=true make test-e2e-release
+      resources:
+        requests:
+          cpu: "4"
+          memory: "8Gi"
+        limits:
+          cpu: "4"
+          memory: "8Gi"
+    nodeSelector:
+      kubernetes.io/arch: amd64
+
+- name: ci-etcd-unit-test-release36-amd64
```

Comment:
We might not need to run unit test periodically. Usually unit test doesn't have any randomness, running it in each PR should be already enough. But it isn't a big problem.

Reply:
Should we remove then the unit tests altogether (AMD and ARM)? cc. @serathius

Reply:
Not understanding the argument, unit tests can still have flakes due to races. Are we basing this on unit tests just being more stable than other types of test?